client/src/components/pages/Subject/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import Button from '../../common/Button';
+
+import('./index.css');
+
+const Subject = ({ subject = 'Arabic' }) => {
+  return (
+    <React.Fragment>
+      <div className="Student__div">
```

Comment:
some remarks on CSS namings. I suggest using [BEM](http://getbem.com/introduction/), which has the following format: block__element--modifier (modifier is optional).
The top level element is the block - thus the class name here should be just `subject` (and not 'student' ;) also note the styles name should be either camelCase or kebab-case, ie. first letter lower case

Reply:
thanks for the review